services/121-service/package.json

```diff
@@ -67,7 +67,8 @@
     "eslint-plugin-prettier": "^3.1.4",
     "jest": "^24.9.0",
     "prettier": "^1.18.2",
+    "ts-jest": "^24.0.0",
     "tsc-watch": "^2.4.0",
-    "ts-jest": "^24.0.0"
+    "typedoc": "^0.19.2"
```

Comment:
Are these changes (and in `package-lock.json`) generated via the command run on the Docker-environment?

So via `docker-compose exec 121-service npm install typedoc --save-dev`?
Otherwise the dependencies-of-this-dependency might be specific for your platform (it could be the same in the end, but the whole point of the Docker-env is predictability and reproducibility...)

Reply:
I would suggest to:

- [ ] add the `docs/`-folder to the `.gitignore`-file (so nothing of these files need to be committed)
- [ ] Add a "npm-script" to this file (in the "scripts"-block above) like:
  ```
  "docs:generate": "typedoc --out docs src",
  ```
- [ ] Add a section to the README.md file for this service (or in the services/README, as we want to enable this in all services, I guess.)  
  In it we could explain why you want to generate this and when/how you can look at it. Including that you need to open some specific file/URL in your browser etc...

Reply:
You can make new commits to this branch, but maybe better to "amend"  the previous commit and "force-push" that new commit(s) to this branch.

That way the git-history will not be filled with all these (useless) changes to all these files; as it will only occupy space and make the repository slower over time.